Lib/test/test_pyclbr.py

```diff
@@ -214,6 +220,18 @@ def compare(parent1, children1, parent2, children2):
 
         compare(None, actual, None, expected)
 
+    def test_pdb_module(self):
+        import importlib.util
```

Comment:
This can be a top-level import, no need for a local one (we don't really need fast start up time for tests)